lua/astrocommunity/motion/mini-jump/init.lua

```diff
@@ -0,0 +1,13 @@
+return {
+  "echasnovski/mini.jump",
+  event = "User AstroFile",
+  opts = {
+    delay = {
+      -- Delay between jump and highlighting all possible jumps
+      highlight = 250,
+
+      -- Delay between jump and automatic stop if idle (no jump is done)
+      idle_stop = 1000,
+    },
+  },
```

Comment:
```suggestion
  opts = {},
```

Reply:
I dont see any reason to change the defaults for now. lets wait on feedback from community before we add in some defaults.